src/util/log.js

```diff
@@ -140,9 +140,11 @@ function getLogger () {
 // child logger that has an additional reqID correlation parameter) if we're
 // called in the context of a request.  Otherwise, pass the call onto the default
 // logger.
+
+// How do we get the reqest here
 module.exports = new Proxy (logger, {
-    get (target, key, receiver) {
-        const logger = getLogger();
+    get (target, key, receiver, req) {
+        const logger = getLogger(req);
```

Comment:
Still need to figure this out..... I don't think`req` is not being passed correctly because I can't figure out where/how it's being called. I think somewhere in `src/routes.js`. Also, `req` is being used a lot in `src/routes.js` and I want to figure out exactly how that works and where it's getting req from...